src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -87,14 +45,31 @@ public IndexModel describeIndex(String indexName) throws PineconeException {
         return indexModel;
     }
 
-    public IndexModel configureIndex(String indexName, ConfigureIndexRequest configureIndexRequest) throws PineconeValidationException {
-        if (configureIndexRequest == null) {
-            throw new PineconeValidationException("ConfigureIndexRequest object cannot be null");
-        }
+    public IndexModel configureIndex(String indexName, String podType, Integer replicas) throws PineconeValidationException {
         if (indexName == null || indexName.isEmpty()) {
-            throw new PineconeValidationException("Index name cannot be null or empty");
+            throw new PineconeValidationException("indexName cannot be null or empty");
+        }
+
+        if (podType == null && replicas == null) {
```

Comment:
Any reason why we dont want to check for empty string values for podtype and replicas?